MvvmCross.Tests/MvxUnitTestCommandHelper.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using MvvmCross.Commands;
+
+namespace MvvmCross.Tests
+{
+    public class MvxUnitTestCommandHelper : IMvxCommandHelper
+    {
+        public event EventHandler CanExecuteChanged;
+
+        private Dictionary<object, int> items = new Dictionary<object, int>();
+
+        public void WillCallRaisePropertyChangedFor(object item)
+        {
+            items.Add(item, 0);
+        }
+
+        public bool HasCalledRaisePropertyChangedFor(object item)
+        {
+            try
+            {
+                return items[item] > 0;
```

Comment:
There should be protection in here for when the item doesn't exist, rather than just allowing it to throw an exception